jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java

```diff
@@ -69,6 +69,36 @@ public void setUp() throws Exception {
     );
   }
 
+
+  @Test
+  public void testForParsePropertyKey() throws IOException {
+    JDBCInterpreter t = new JDBCInterpreter(new Properties());
+    
+    // if return null is that prefix not found
+    assertEquals(t.getPropertyKey("(fake) select max(cant) from test_table where id >= 2452640"),
+        null);
```

Comment:
@vgmartinez I saw the failure and found out the below:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.apache.zeppelin.jdbc.JDBCInterpreterTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/travis/build/apache/incubator-zeppelin/zeppelin-interpreter/target/zeppelin-interpreter-0.6.0-incubating-SNAPSHOT.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/travis/build/apache/incubator-zeppelin/zeppelin-interpreter/target/lib/slf4j-log4j12-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/travis/.m2/repository/org/slf4j/slf4j-log4j12/1.7.10/slf4j-log4j12-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
log4j:WARN No appenders could be found for logger (org.apache.zeppelin.jdbc.JDBCInterpreter).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.474 sec <<< FAILURE! - in org.apache.zeppelin.jdbc.JDBCInterpreterTest
testForParsePropertyKey(org.apache.zeppelin.jdbc.JDBCInterpreterTest)  Time elapsed: 0.339 sec  <<< FAILURE!
java.lang.AssertionError: expected:<fake> but was:<null>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.apache.zeppelin.jdbc.JDBCInterpreterTest.testForParsePropertyKey(JDBCInterpreterTest.java:78)
```
It needs to fix the test case. It should return `fake` actually.